src/shared/styles/base.scss

```diff
@@ -72,3 +72,14 @@ h6 {
     padding: 0 toRem(20);
   }
 }
+.Toastify__progress-bar--wrp {
+  width: 0 !important;
+  overflow: hidden;
+}
+.Toastify__toast-theme--light {
+  background: #dedede !important;
+  color: #757575;
+}
+.Toastify__toast-container--top-right {
+  top: 35% !important;
+}
```

Comment:
move it to separate file into module of this component
base file is used for general styles